pkg/flowaggregator/flowaggregator.go

```diff
@@ -15,64 +15,240 @@
 package flowaggregator
 
 import (
+	"fmt"
+	"hash/fnv"
 	"net"
 	"time"
 
-	"github.com/vmware/go-ipfix/pkg/collector"
-	"github.com/vmware/go-ipfix/pkg/entities"
-	"github.com/vmware/go-ipfix/pkg/exporter"
-	"github.com/vmware/go-ipfix/pkg/registry"
+	ipfixentities "github.com/vmware/go-ipfix/pkg/entities"
+	ipfixintermediate "github.com/vmware/go-ipfix/pkg/intermediate"
+	ipfixregistry "github.com/vmware/go-ipfix/pkg/registry"
 	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/ipfix"
+	"github.com/vmware-tanzu/antrea/pkg/util/env"
+)
+
+var (
+	IANAInfoElements = []string{
+		"flowStartSeconds",
+		"flowEndSeconds",
+		"sourceIPv4Address",
+		"destinationIPv4Address",
+		"sourceTransportPort",
+		"destinationTransportPort",
+		"protocolIdentifier",
+		"packetTotalCount",
+		"octetTotalCount",
+		"packetDeltaCount",
+		"octetDeltaCount",
+		"originalExporterIPv4Address",
+		"originalObservationDomainId",
+	}
+	IANAReverseInfoElements = []string{
+		"reversePacketTotalCount",
+		"reverseOctetTotalCount",
+		"reversePacketDeltaCount",
+		"reverseOctetDeltaCount",
+	}
+	AntreaInfoElements = []string{
+		"sourcePodName",
+		"sourcePodNamespace",
+		"sourceNodeName",
+		"destinationPodName",
+		"destinationPodNamespace",
+		"destinationNodeName",
+		"destinationClusterIPv4",
+		"destinationServicePortName",
+	}
+	aggregationFields = []string{
```

Comment:
We should add destinationClusterIP and destinationServicePortName for K8s service info. Are these not required as base record for correlation is from source node in go-ipfix intermediate process? 
We should also add ingressNetworkPolicyName and ingressNetworkPolicyName for correlation fields.